src/modules/form/data/search.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "search",
+  "item": [
+    {
+      "type": "string",
+      "required": false,
+      "linkId": "1",
+      "text": "Personal ID",
+      "mapping": {
+        "clientSearchInput": "personalId"
+      }
+    },
+    {
+      "type": "string",
+      "required": false,
+      "linkId": "2",
+      "text": "First Name",
+      "mapping": {
+        "clientSearchInput": "firstName"
+      }
+    },
+    {
+      "type": "string",
+      "required": false,
+      "linkId": "3",
+      "text": "Last Name",
+      "mapping": {
+        "clientSearchInput": "lastName"
+      }
+    },
+    {
+      "type": "string",
+      "required": false,
+      "linkId": "4",
+      "text": "Last 4 Social",
+      "mapping": {
+        "clientSearchInput": "ssnSerial"
+      }
```

Comment:
Experimenting with a "mapping" field here to map fields into a ClientSearchInput object, for the "Advanced Search" section of the client search page. Will definitely need to come back to this when figuring out how to submit the client creation form and assessments